### PR TITLE
[2201.9.2] Fix WSDL release-note to address review comments.

### PR DESCRIPTION
**Pull request by AzeemMuzammil**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

**Review by ayeshLK — APPROVED** [review]
LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.9.2/RELEASE_NOTE.md [hunk 1]
```diff
@@ -50,16 +50,16 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.2)](
 
 ### Improvements
 
-Introduced the WSDL CLI-tool as an experimental feature which generates Ballerina types from a WSDL file. This tool can be accessed by pulling it from Ballerina Central and running it with an input WSDL file.
+Introduced the WSDL CLI tool as an experimental feature which generates Ballerina types from a WSDL file. This tool can be accessed by pulling it from Ballerina Central and running it with an input WSDL file.
```
Comment (NipunaRanasinghe) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.2/RELEASE_NOTE.md:53]: It seems we have added this new tool under the `improvements` section. Not sure if this should come under `New Features` instead. 

IIRC, sometimes back we agreed thats its okay to ship tooling features in patch releases. So mentioning it under new features should be fine IMO. @anupama-pathirage @keizer619 wdyt?

Comment (AzeemMuzammil) [reply downloads/swan-lake-release-notes/swan-lake-2201.9.2/RELEASE_NOTE.md:53]: +1 to move under new feature,

File: downloads/swan-lake-release-notes/swan-lake-2201.9.2/RELEASE_NOTE.md [hunk 2]
```diff
@@ -48,18 +48,20 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.2)](
 
 ## Developer tools updates
 
-### Improvements
+### New features
+
+#### WSDL tool
 
-Introduced the WSDL CLI-tool as an experimental feature which generates Ballerina types from a WSDL file. This tool can be accessed by pulling it from Ballerina Central and running it with an input WSDL file.
+Introduced the WSDL CLI tool as an experimental feature which generates Ballerina types from a WSDL file. This tool can be accessed by pulling it from Ballerina Central and running it with an input WSDL file.
```
Comment (azinneera) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.2/RELEASE_NOTE.md:55]:
```suggestion
Introduced the WSDL CLI tool as an experimental feature that generates Ballerina types from a WSDL file. This tool can be accessed by pulling it from the Ballerina Central and running it with an input WSDL file.
```

**Review by NipunaMadhushan — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.9.2/RELEASE_NOTE.md (hunk 1 above)
Comment (anupama-pathirage) [reply downloads/swan-lake-release-notes/swan-lake-2201.9.2/RELEASE_NOTE.md:55]: +1 to move it to new feature

**Pull request merged**